### DISCUSSION TITLE
Add X (used to be twitter) icon.

### DISCUSSION THREAD

**Discussion by BladeRunner713** [react-icons/react-icons#787] [New Icon Set]
Since twitter changed its logo, we need new X logo in our websites.

**Comment by mvaibhav77** [comment]
**The Twitter X logo, already present in Font Awesome but missing from React-Icons, should be promptly integrated into the react-icons library.**

**Comment by kamijin-fanta** [comment]
resolved in https://github.com/react-icons/react-icons/pull/807 :tada:

**Comment by lalitkumawat1m** [comment]
Still Facing Error while using 4.11.0 version

**Reply by kamijin-fanta** [reply]
you can try it by write `import {FaXTwitter} from 'react-icons/fa6';`.

**Reply by lalitkumawat1m** [reply]
Thank you so much it works

**Reply by vinayakhegde1** [reply]
@kamijin-fanta thanks for this i was searching for it 👍

**Reply by tejas-2232** [reply]
How to use The New X icon in HTML?

**Reply by Hasanelmestrah** [reply]
> you can try it by write `import {FaXTwitter} from 'react-icons/fa6';`.

thank you it work fine

**Reply by edson-magombo** [reply]
thank you bro

**Comment by eliotgevers** [comment]
`import {FaXTwitter} from 'react-icons/fa6`

worked for me

**Reply by fastodigama** [reply]
Thanks :)
